python/valis/routes/query.py

```diff
@@ -8,6 +8,7 @@
 from fastapi_restful.cbv import cbv
 from pydantic import BaseModel, Field, BeforeValidator
 
+from valis.cache import CACHE_TTL, valis_cache
```

Comment:
is `CACHE_TTL` used in this module?  If not, maybe we can remove this?

Reply:
Yes, removed.